addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/internal/FreeboxHandlerFactory.java

```diff
@@ -38,12 +47,28 @@
  */
 public class FreeboxHandlerFactory extends BaseThingHandlerFactory {
 
+    private final Logger logger = LoggerFactory.getLogger(FreeboxHandlerFactory.class);
+
     private Map<ThingUID, ServiceRegistration<?>> discoveryServiceRegs = new HashMap<>();
 
+    private AudioHTTPServer audioHTTPServer;
+
+    private Map<ThingUID, ServiceRegistration<AudioSink>> audioSinkRegistrations = new ConcurrentHashMap<>();
+
+    // url (scheme+server+port) to use for playing notification sounds
+    private String callbackUrl = null;
```

Comment:
Ok.
Should be changed in other bindings using audio sink like the Sonos binding.